src/PerformanceTests/PTODBCExecution/performance_odbc_execution.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright <2019> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+// clang-format off
+#include "chrono"
+#include <es_odbc.h>
+#include <string>
+#include <vector>
+#include <iostream>
+// clang-format on
+#define IT_SIZEOF(x) (NULL == (x) ? 0 : (sizeof((x)) / sizeof((x)[0])))
+#define ITERATION_COUNT 12
+std::wstring dsn_name = L"DSN=test_dsn";
+const wchar_t* const query = L"SELECT * FROM kibana_sample_data_flights limit 10000";
+
+int main() {
+    SQLTCHAR out_conn_string[1024];
+    SQLSMALLINT out_conn_string_length;
+    SQLHENV env = SQL_NULL_HENV;
+    SQLHDBC conn = SQL_NULL_HDBC;
+    SQLHSTMT hstmt = SQL_NULL_HSTMT;
+    SQLRETURN ret = SQL_ERROR;
+    try {
+        if (SQL_SUCCEEDED(SQLAllocHandle(SQL_HANDLE_ENV, SQL_NULL_HANDLE, &env))
```

Comment:
Let's pull these initialization steps out into a separate function, since they are done before we start timing

Reply:
done